lib/utility/_query.js

```diff
@@ -39,6 +39,9 @@ exports._query = function _query(query, data, options, callback, executor) {
 		// Use resultSet if 'useResultSet' option is set
 		queryOptions.resultSet = true;
 		queryOptions.prefetchRows = prefetchRows;
+	} else if (options && options.write) {
+		// Save changes to the db
+		queryOptions.saveChanges = true;
```

Comment:
I definitely will use it instead of another option param, I just somehow have missed it :)